app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,27 @@
+class UserMailer < ApplicationMailer
+
+  # Subject can be set in your I18n file at config/locales/en.yml
+  # with the following lookup:
+  #
+  #   en.user_mailer.confirmation.subject
+  #
+  default from: User::MAILER_FROM_EMAIL
```

Comment:
we do not need to use a constant here, and not from user model. if you want to create a constant create here or in application mailer

Reply:
okay sir